scripts/build_utils.sh

```diff
@@ -510,8 +510,8 @@ update-alternatives \
 update-alternatives --auto gcc
 
 # cmake uses the latter by default
-ln -nsf /usr/bin/gcc /usr/bin/x86_64-linux-gnu-gcc
-ln -nsf /usr/bin/g++ /usr/bin/x86_64-linux-gnu-g++
+ln -nsf /usr/bin/gcc /usr/bin/\$(arch)-linux-gnu-gcc
```

Comment:
where is `$(arch)` coming from? I can't find where it is defined in this file

Reply:
please note, `$(arch)` != `${arch}`.